### PR TITLE
kweirauch/error pipeline

### PR DESCRIPTION
**Pull request by Korosensei42**
Just changed the code and severity to only include the relevant attributes.

### CODE REVIEW

**Comment by Korosensei42** [comment]
Changed targetbranch

File: src/vocabularyChecker/detectWrongSyntax.test.ts [hunk 1]
```diff
@@ -62,13 +62,13 @@ describe("logSingleWord", () => {
 		const position: Position = { fromIndex: 0, toIndex: 4 };
 		const issues = logSingleWord("Hello", position);
 		expect(issues).toEqual({
-			code: InvalidWord,
+			code: InvalidWord.INVALID_WORD.message,
 			message: `${"Hello"} von Stelle ${position.fromIndex} bis ${position.toIndex} ist ein unerlaubtes Wort! \n`,
```
Comment (Khazuar) [inline src/vocabularyChecker/detectWrongSyntax.test.ts]: This is not the message defined in the JSON for this Issue.

**Comment by Khazuar** [comment]
@Korosensei42 Tests are still failing.

File: src/vocabularyChecker/detectWrongSyntax.test.ts (hunk 1 above)
Comment (Korosensei42) [reply src/vocabularyChecker/detectWrongSyntax.test.ts]: What is the right message then, if not InvalidWord.INVALID_WORD.message ?

**Comment by Korosensei42** [comment]
> @Korosensei42 Tests are still failing.

For me, only the Prolog-tests are failing, which they did before.

**Pull request merged**